packages/react-devtools-shared/src/devtools/views/Components/InspectedElementView.js

```diff
@@ -71,9 +72,14 @@ export default function InspectedElementView({
     rendererPackageName !== null && rendererVersion !== null
       ? `${rendererPackageName}@${rendererVersion}`
       : null;
+  const reconcilerLabel =
+    reconcilerVersion !== null ? `react-reconciler@${reconcilerVersion}` : null;
```

Comment:
That may be confusing since this looks like an npm package specifier. But the `react-reconciler` version on npm is independent of react (`0.26.2`).

Reply:
Yeah that's a good point. I should just not display the reconciler-injected version.